homeworks/denis.yeromenko_Denis-Yeromenko/homework_7/fibonachi.js

```diff
@@ -0,0 +1,12 @@
+const numFibonachi = () => {
+    const lastNumb = Math.floor(Math.random() * (100 - 50 + 1)) + 50;
+    const arr = [0, 1];
+    for (let i = 0; i < 10; i++) {
```

Comment:
limiting the number of iterations is wrong, you need a solution that will be pushing items to array while they are still less than provided max value